src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1349,4 +1349,7 @@ Please specify one of them using the `--source` option to proceed.</value>
   <data name="ShowLabelInstallationNotes" xml:space="preserve">
     <value>InstallationNotes:</value>
   </data>
+  <data name="SymlinkModified" xml:space="preserve">
+    <value>Portable symlink not deleted as it was modified and points to a different target exe</value>

```

Comment:
Making loc changes after we have begun the release process makes it take longer.  We should discuss whether this change makes it to 1.3 today with @denelon.

Reply:
Discussed with @denelon, this will be included in 1.4